.travis.yml

```diff
@@ -5,30 +5,9 @@ sudo: required
 matrix:
   fast_finish: true
   include:
-    - name: "PHP 5.5 / MySQL 5.6"
-      php: "5.5"
-      dist: trusty
-    - name: "PHP 5.6 / MySQL 5.6"
-      php: "5.6"
-      dist: trusty
-    - name: "PHP 7.0 / MySQL 5.7"
```

Comment:
I wonder if we should actually just remove PHP 7.0. Is there really much benefit from testing 7.0 alongside 7.1?

I guess ideally I'd like to split things up so we maybe do a daily build that runs through absolutely everything while still having a fairly speedy process for PRs.

Reply:
I think Travis has some form of scheduled jobs that'd allow this? I certainly wouldn't be opposed.

Can we remove 5.5 while we're at it ;)